### ISSUE TITLE
publication on Zenodo

### ISSUE THREAD

**Issue by Mestyan** [ProjectJaraid/jaraid_source#80]
Dear @tillgrallert you suggested formally publishing the dataset with a version number and an unambiguous identifier on platforms such as Zenodo - would you do that? Then I can add the citation to the "how to refer" section
I would start to advertise on 15 November - I want to do some more work on 
1) data cleaning and 
2) visualizations, 
3) prepare an Arabic version (if you can do an Arabic quasi-xml file with your XSLT code transformation I can start to clean/handle that

**Comment by tillgrallert** [comment]
Dear @Mestyan, yes, I can do so. There are basically two ways of approaching this issue:

1. Connect GitHub and Zenodo account and set Zenodo to watch a specific repository for releases. As soon as a release is published on Github, it gets archived at and a DOI from Zenodo. Metadata for the release can be independently edited on Zenodo at any time. Advantages are clear but there is one minor nuisance, one will not know the DOI before the first release of a repository.
2. Upload directly to Zenodo. This allows to reserve a DOI before publication. The disadvantage is that one has to do everything manually.

**Comment by tillgrallert** [comment]
Regarding the final question, I will answer in the separate issue

**Comment by Mestyan** [comment]
great, thanks @tillgrallert option 1 sounds great. The question is what "release" means in Github? You mean a merge?

**Comment by tillgrallert** [comment]
Github is a software development platform and therefore, one can release the code in a repository. The common approach is to create a new branch, customarily called `rc` (release candidate), draft a release from this branch, check that everything is as expected, and then release the code with a semantic version number, such as v0.1, v1.0, v1.0.5 etc.

**Comment by tillgrallert** [comment]
I suggest to create a small empty release to get a DOI from Zenodo and then prepare something more proper in the coming days. What do you think?

**Comment by Mestyan** [comment]
Thank you so much! OK, though I still do not know what "draft a release" means exactly but I will educate myself. and Zenado is watching that branch.

**Comment by Mestyan** [comment]
Yes, I agree

**Comment by Mestyan** [comment]
I will create an r c brancj tonight in jaraid_source = this is the publication, and not the website "projectjaraid" , right? The data in the xml file is the publication, correct?

**Comment by tillgrallert** [comment]
Yes, the release concerns the data, not the website. I added an annotated screenshot for where to look for creating new releases.

<img width="1414" alt="Screenshot 2020-11-04 at 13 52 46" src="https://user-images.githubusercontent.com/5872574/98114223-360a1680-1ea5-11eb-9d94-b5fdb35a8ef0.png">

**Comment by Mestyan** [comment]
OK, I am an idiot , I should have seen this 1000 times, thanks

**Comment by Mestyan** [comment]
created the release as pre-release, can you see it?

**Comment by Mestyan** [comment]
@tillgrallert I do not know how to connect Zenodo to the rc branch release... sorry can you connect them? thanks

**Comment by tillgrallert** [comment]
@Mestyan I have just toggled the switch for archiving releases on Zenodo. But it is a bit weird that you cannot do so. Both of our accounts are registered as "admin" for this repository and should therefore both be able to do once we have linked our GitHub accounts on Zenodo. Be that as it may, everything is now ready. As soon as we publish a release, we will get a DOI from Zenodo and can edit the metadata there to our liking.

**Comment by tillgrallert** [comment]
Dear @Mestyan,
I just made a new and completely empty pre-release to reserve a DOI from Zenodo. The DOI to cite all versions is [10.5281/zenodo.4264985](https://doi.org/10.5281/zenodo.4264985) it will take a short while to propagate through the internet.

**Comment by Mestyan** [comment]
great, thank you, I will check this later

**Comment by Mestyan** [comment]
I think we should change the title of the dataset or the title of the publication as it is published as "Project Jarāʾid" but Jarāʾid is only the main title. To be discussed

**Comment by Mestyan** [comment]
@tillgrallert shall we do a version 1 release? Or you want to wait more?

**Comment by tillgrallert** [comment]
Dear Adam, I think we can do a first release. I am very much in favour of a) being able to provide an authoritative citation for the data set and b) frequent releases. We should probably clean up the repository a bit before the initial release and improve the documentation.

# clean-up

1. The actions can be cleaned up:
     - they should make use of XSLT from other repositories using the respective gh-pages branch
2. the `html/` folder: is this needed?

# documentation

1. I will rewrite the readme.md, explaining the folder structure and the current data model(s).
2. I will add a licence.md stating the license of our data as CC BY-NC 4.0

**Comment by tillgrallert** [comment]
I just added `license.md`, updated `readme.md`, and changed the action for updating Arabic columns to make use of XSLT from jaraid_tools.

**Comment by Mestyan** [comment]
I deleted the html folder as the action delivers them to the website repository

**Comment by Mestyan** [comment]
I see many small issues to correct in both the master and the authority files but I think this is endless work so if you think so then let's do a v1 release. Also, important: I think the title should be  Adam Mestyan and Till Grallert **et al.** (2020). ~~Project~~ Jarāʾid: A Chronology of Arabic Periodicals (1800-1929) (Version vXXXX) [Data set]. Zenodo. http://doi.org/10.5281/zenodo.4264986.

**Comment by Mestyan** [comment]
@tillgrallert the TEI master file already contains reference to the Creative Commons copyright of Attribution-NonCommercial 4.0 International (CC BY-NC 4.0)

**Comment by Mestyan** [comment]
Dear @tillgrallert would you do the publication please? Only you can do so in Zenodo! Thanks

**Comment by tillgrallert** [comment]
Dear @Mestyan. Sure. I will do so by merging the `master` branch into `rc` and then draft a release of the `rc` branch. I would ask you to check my draft of the release notes before the actual release.
 Once this is done, I will update the bibliographic information on Zenodo.

**Comment by Mestyan** [comment]
I can do the merge! I am doing it now to help !

**Comment by tillgrallert** [comment]
Please wait a second. I am cleaning some technical stuff in the authority file

**Comment by Mestyan** [comment]
Ah, sorry, OK, just let me know

**Comment by tillgrallert** [comment]
I will :-)

**Comment by tillgrallert** [comment]
Dear @Mestyan, I suggest we remove all the github actions and XSLT from `rc`. What do you think?

**Comment by Mestyan** [comment]
yes, absolutely, I thought we only publish the master file as a dataset

**Comment by Mestyan** [comment]
+ authority file

**Comment by Mestyan** [comment]
I mean and authority file

**Comment by tillgrallert** [comment]
Perfect. had the same thought :-)

**Comment by Mestyan** [comment]
OK so should I do the merge? Or you do it?

**Comment by tillgrallert** [comment]
Just did so and I am now drafting the release

**Comment by Mestyan** [comment]
cool, thanks!

**Comment by tillgrallert** [comment]
Please check out the short release note on the draft

**Comment by Mestyan** [comment]
I edited a tiny bit but I think it is perfect

**Comment by tillgrallert** [comment]
So let's release it

**Comment by Mestyan** [comment]
Yalla ya ustadh!

**Comment by tillgrallert** [comment]
done

**Comment by Mestyan** [comment]
I think the citation reference  should be changed to: 
Adam Mestyan and Till Grallert et al. (2020). Project Jarāʾid: A Chronology of Arabic Periodicals (1800-1929) (Version vXXXX) [Data set]. Zenodo.

Sorry only you can change it in Zenodo

**Comment by tillgrallert** [comment]
Already at it.

(Alif mabrouk and thank you so much for your gargantuan effort of transcribing the entire Tarrazi)

**Comment by Mestyan** [comment]
Mabrouk to you and thank you for the patience and help!

**Comment by Mestyan** [comment]
I see the change. Fantastic, great ! I will advertise on Facebook and Twitter pls do so as well . Great job between you me and Hugh! Thanks for the teaching!
